modules/renderer/background_source.js

```diff
@@ -312,6 +307,9 @@ rendererBackgroundSource.Esri = function(data) {
 
         if (inflight[tileId]) return;
 
+        // instead of calling fetchTilemap when the metadata window is open, we should call it when editing is first activated
+        fetchTilemap(center, esri);
```

Comment:
> switch the `const/let` back to old-school `var`

👍 

> what is missing from this now?

right now `fetchTilemap()` is only called when the background metadata window is opened w/ <kbd>⌘ + Shift + B</kbd>.

i'm not sure how/where, but the method needs to be called as soon as users cross the zoom threshold to put `iD` into 'edit mode' (when Esri imagery is being used in the background anyway).